### PR TITLE
[broadcom]: Update Broadcom debian packages

### PR DESCRIPTION
**Pull request by stcheng**
- Support Tomahawk BCM56963
- Add new config.bcm files

Signed-off-by: Shuotian Cheng <shuche@microsoft.com>

### CODE REVIEW

**Comment by stcheng** [comment]
I'll re-submit this pull request later to have a different version number.

File: device/arista/x86_64-arista_7060_cx32s/Arista-7060-CX32S/sai.profile [hunk 1]
```diff
@@ -1 +1 @@
-SAI_INIT_CONFIG_FILE=/etc/bcm/th-a7060-cx32s-32x100G.config.bcm 
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-a7060-32x100G.config.bcm
```
Comment (lguohan) [inline device/arista/x86_64-arista_7060_cx32s/Arista-7060-CX32S/sai.profile]: I prefer to keep cx32s and qx32, as there are a7050-qx32s as a different sku.

**Comment by stcheng** [comment]
@lguohan updated.

**Comment by lguohan** [comment]
have you  also updated the sai package to change the config bcm names to be the same as the one in sai.profile?

**Comment by stcheng** [comment]
@lguohan yes.

**Comment by lguohan** [comment]
thanks.

**Pull request merged**